int32 UNLABELED=0
int32 BINARY_LABEL=1
int32 COST_LABEL=2
int32 type
task_recorder/BinaryLabel binary_label
task_recorder/CostLabel cost_label
